src/core/PathUtil.cpp

```diff
@@ -200,4 +200,22 @@ namespace lmms::PathUtil
 		return basePrefix(shortestBase) + relativeOrAbsolute(absolutePath, shortestBase);
 	}
 
+	std::filesystem::path LMMS_EXPORT fsConvert(const QString& path)
```

Comment:
```suggestion
	std::filesystem::path fsConvert(const QString& path)
```
`LMMS_EXPORT` only needs to go in the header